### ISSUE TITLE
Remove mentions of 'unicode' type from twisted.logger._format

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10001]
|[<img alt="mthuurne's avatar" src="https://avatars.githubusercontent.com/u/246676?s=50" width="50" height="50">](https://github.com/mthuurne)| @mthuurne reported|
|-|-|
|Trac ID|trac#10001|
|Type|defect|
|Created|2020-10-06 00:06:54Z|

When running pydoctor on Twisted, it outputs:
```
src/twisted/logger/_format.py:204: invalid ref to 'unicode.format' not resolved
src/twisted/logger/_format.py:204: invalid ref to 'unicode.format' not resolved
```
Besides these two instances of `unicode` causing warnings, there are a bunch more that cause less of a fuss but are nevertheless incorrect in Python 3.

<details><summary>Searchable metadata</summary>

```
trac-id__10001 10001
type__defect defect
reporter__mthuurne mthuurne
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__logger logger
keywords__None None
time__1601942814254772 1601942814254772
changetime__1602044574876583 1602044574876583
version__None None
owner__None None

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="mthuurne's avatar" src="https://avatars.githubusercontent.com/u/246676?s=50" width="50" height="50">](https://github.com/mthuurne)<a name="note_1"></a>|@mthuurne commented|
|-|-|

PR is here: https://github.com/twisted/twisted/pull/1425

**Comment by twisted-trac** [comment]
|[<img alt="mthuurne's avatar" src="https://avatars.githubusercontent.com/u/246676?s=50" width="50" height="50">](https://github.com/mthuurne)<a name="note_2"></a>|@mthuurne set status to `closed`|
|-|-|

This issue will be fixed as part of [#9995](https://github.com/twisted/twisted/issues/9995).
